app/representers/concerns/uri_methods.rb

```diff
@@ -25,8 +25,8 @@ def prx_model_uri(*args)
   end
 
   def joined_names(args)
-    ( Array(args.collect{|arg| prx_model_uri_part_to_string(arg)}) + 
-      prx_model_uri_suffix(args) ).flatten.compact.join("/")
+    (Array(args.map { |arg| prx_model_uri_part_to_string(arg) } ) +
+      prx_model_uri_suffix(args)).flatten.compact.join("/")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.